src/algorithms/digi/PhotoMultiplierHitDigiConfig.h

```diff
@@ -17,6 +20,11 @@ namespace eicrecon {
       double pedMean       = 200.0;
       double pedError      = 3.0;
 
+      // noise
+      bool enableNoise       = false;
```

Comment:
`rate==0` will certainly disable the noise injection, but I kind of like having this boolean so that we can more easily turn it on or off without adjusting the rate. We can certainly move to an `enum` if we have more noise models, but for now I vote we keep it simple.

Reply:
Sounds good to me.